MediaBrowser.WebDashboard/Api/DashboardService.cs

```diff
@@ -18,7 +23,7 @@
 namespace MediaBrowser.WebDashboard.Api
 {
     /// <summary>
-    /// Class GetDashboardConfigurationPages
+    /// Class GetDashboardConfigurationPages.
```

Comment:
It feels like it would be better to just remove documentation like this entirely instead of having a "fake" summary. I know these are everywhere in the codebase already though so maybe a project for another day

Reply:
Yeah, one thing at a time ;)